crates/core/types/block.rs

```diff
@@ -150,7 +150,7 @@ impl RLPDecode for BlockHeader {
         let (prev_randao, decoder) = decoder.decode_field("prev_randao")?;
         let (nonce, decoder) = decoder.decode_field("nonce")?;
         let nonce = u64::from_be_bytes(nonce);
-        let (base_fee_per_gas, decoder) = decoder.decode_field("base_fee_per_gas")?;
```

Comment:
Hmm, this is not really optional in post-merge networks.

Reply:
Do we want to eventually sync from genesis? I guess we will receive the first blocks without this field 🤔. With this fix I can run the eth suits of tests.